README.md

```diff
@@ -17,7 +17,44 @@ The datapoints for the metrics are individually simulated using a distribution t
 
 ## Getting Started
 
-Settings:
+### Building
+
+metricsgenreceiver is a receiver for the otel collector. To build the otelcollector, the tool
+[ocb](https://opentelemetry.io/docs/collector/custom-collector/) is needed. To install it on OS X, run
+
+```bash
+curl --proto '=https' --tlsv1.2 -fL -o ocb \
+https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/cmd%2Fbuilder%2Fv0.123.0/ocb_0.123.0_darwin_arm64
+chmod +x ocb
+```
+
+Be aware, currently this exact version is needed (0.123.0).
```

Comment:
nit, I can see this easily missed when we bump versions.

Reply:
Yep, we have it in quite a few places. I would skip it for now if that is ok for you and fix it later.